### PR TITLE
:art: :lipstick: Video input block redesign block redesign #644

### LINKED ISSUE #644: :art: :lipstick:  Video Input Block Redesign: block redesign
## User stories

As a user, I want the "Video Input Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/video-input-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/video-input-block/`.

### PR DESCRIPTION
**Pull request by thadeusdev**
# Description

- Replaced video icon with svg
- Replaced menu ellipsis icon with svg
- styled the header
- styled the input field and text
- Re-positioned jsPlumb

Fixes #644

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-11-03 01-06-28

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] My changes generate no new warnings

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-input-block.jsplumb.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function _VideoInputBlockDecoratePlumb(block: VideoInputBlock, comp: Comp
     target: true,
     cssClass:"block_endpoint",
     endpoint: "Dot",
-    anchor: "Left",
+    anchor: [0, 0.2, 0, 0],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-input-block.jsplumb.ts:24]: have this as "Left"

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync and resolve conflicts

**Comment by thadeusdev** [comment]
Hi, I have the requested changes. Thank you for review

**Comment by LemmyMwaura** [comment]
Closing this as its been resolved but good work

**Comment by thadeusdev** [comment]
Noted.

**Pull request closed without merging**